apps/test/unit/code-studio/levels/freeResponse.js

```diff
@@ -0,0 +1,43 @@
+import { expect } from 'chai';
+import FreeResponse from '@cdo/apps/code-studio/levels/freeResponse';
+import { writeSourceForLevel } from '@cdo/apps/code-studio/clientState';
+
+describe('Free Response', () => {
+  const levelId = 1047;
+  const scriptName = 'test-script';
+  const lastAttemptString = 'This is my final answer';
+  const otherLastAttemptString = 'This is some other answer';
+  let textarea;
+
+  before(() => {
+    textarea = document.createElement('textarea');
+    textarea.setAttribute('id', `level_${levelId}`);
+    textarea.setAttribute('class', 'response');
+    document.body.appendChild(textarea);
+    window.appOptions = {};
```

Comment:
I might stash the currently window.appOptions, and then reset it (like here https://github.com/code-dot-org/code-dot-org/blob/staging/apps/test/unit/code-studio/levels/multi.js#L6) to reduce the likelihood that changes to appOptions leak between tests.